pkgs/applications/video/flowblade/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, fetchFromGitHub, stdenv
+, ffmpeg, frei0r, sox, gtk3, python3, ladspaPlugins
+, gobject-introspection, makeWrapper, wrapGAppsHook
+}:
+
+let
+  python_env = python3.withPackages (ps: with ps; [
+    mlt-bindings
+    pygobject3
+    dbus-python
+    numpy
+    pillow
+  ]);
+in
+stdenv.mkDerivation rec {
+  pname = "flowblade";
+  version = "2.8.0.3";
+
+  src = fetchFromGitHub {
+    owner = "jliljebl";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-/EkI3qiceB5eKTVQnpG+z4e6yaE9hDtn6I+iN/J+h/g=";
+  };
+
+  buildInputs = [ ffmpeg frei0r sox gtk3 python_env gobject-introspection ladspaPlugins ];
```

Comment:
```suggestion
  buildInputs = [ ffmpeg frei0r sox gtk3 gobject-introspection ladspaPlugins ]
    ++ python3.withPackages (ps: with ps; [ mlt-bindings pygobject3 dbus-python numpy pillow ]);
```

Reply:
Got an `error: value is a set while a list was expected` here. Added it inside the list directly. Hope that works, too,